src/main/scala/symsim/concrete/ConcreteExactRL.scala

```diff
@@ -33,3 +33,10 @@ trait ConcreteExactRL[State, ObservableState, Action]
   /** Convert the matrix Q after training into a Policy map. */
   def qToPolicy (q: Q) (using Ordering[Double]): Policy =
     q.states.map { s => (s, vf.bestAction (q) (s)) }.to (Map)
+
+  /** Evaluate the policy `p` on `episodes` episodes generated using the
+   *  agent's initialization function.
+   */
+  def policyEval (p: Policy, episodes: Int = 5): Randomized[Double] =
```

Comment:
I will rename this to evaluate as the two functions do the same thing, just for different arguments so it makes sense to overload them.